### PR TITLE
Blog post: Camel Language Server including autocompletion from Kubernetes context

### PR DESCRIPTION
**Pull request by Delawen**

### CODE REVIEW

**Comment by apupier** [comment]
build error:

```
[check:html     ]   1:9252  error  The autoplay attribute is not allowed on <video>  no-autoplay
[check:html     ]   1:9547  error  The autoplay attribute is not allowed on <video>  no-autoplay
```

File: content/blog/2023/10/camel-language-server-kubernetes-context/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "VSCode Language Support Autocompletion from Kubernetes Context"
+date: 2023-10-27
+authors: ["delawen"]
+categories: ["Tooling"]
+preview: "VSCode Language Support Autocompletion from Kubernetes Context"
+summary: "The Camel Language Server is now able to autocomplete configmaps and secrets coming from a Kubernetes connection."
+---
+
+Coming in the next VS Code Language support for Apache Camel, we are introducing a better context based autocompletion. We already had namespaces being auto completed when a related property was filled and there was a kubernetes connection configured.
```
Comment (apupier) [inline content/blog/2023/10/camel-language-server-kubernetes-context/index.md:10]: it is already released :-) In Vs Code language Support 0.12.0

File: content/blog/2023/10/camel-language-server-kubernetes-context/index.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: "VSCode Language Support Autocompletion from Kubernetes Context"
```
Comment (apupier) [inline content/blog/2023/10/camel-language-server-kubernetes-context/index.md:2]:
```suggestion
title: "VS Code Language Support Autocompletion from Kubernetes Context for secrets and configmaps"
```

there was already completion for namespace, so I think we need to be a bit more specific in the title.

VSCode --> VS Code

File: content/blog/2023/10/camel-language-server-kubernetes-context/index.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+categories: ["Tooling"]
+preview: "VSCode Language Support Autocompletion from Kubernetes Context"
+summary: "The Camel Language Server is now able to autocomplete configmaps and secrets coming from a Kubernetes connection."
+---
+
+Coming in the next VS Code Language support for Apache Camel, we are introducing a better context based autocompletion. We already had namespaces being auto completed when a related property was filled and there was a kubernetes connection configured. 
+
+Now the auto completion loads kubernetes context placeholders too as suggestions. On this case, it doesn't discriminate by name of the property or which camel connector we are configuring; it detects when a `{{secret:` or`{{configmap:` is being written as value in camel uri properties.
+
+<video loop="true" autoplay="true" src="https://user-images.githubusercontent.com/726590/273884659-c03020ae-bd5b-4328-a57c-d8aa7750e79f.webm" controls="controls" muted="muted" ></video>
```
Comment (apupier) [inline content/blog/2023/10/camel-language-server-kubernetes-context/index.md:14]: have you tried to build the site locally?
Last time I tried (several months if not years ago, the video were not working)

File: content/blog/2023/10/camel-language-server-kubernetes-context/index.md (hunk 1 above)
Comment (apupier) [inline content/blog/2023/10/camel-language-server-kubernetes-context/index.md:10]:
```suggestion
Coming in the next [VS Code Language support for Apache Camel](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-apache-camel), we are introducing a better context based autocompletion. We already had namespaces being auto completed when a related property was filled and there was a Kubernetes connection configured. 
```

**Review by apupier — CHANGES_REQUESTED** [review]
- the camel-featured.jpeg and the videos are cropping the completion list. As it is the main feature, i think it will provide a better view if it is more centered on it

**Review by oscerd — APPROVED** [review]

**Comment by oscerd** [comment]
There is no point in leaving this open after 5 months. Closing.

**Pull request closed without merging**